ros2pkg/ros2pkg/resource/ament_cmake/package.xml.em

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>@package_name</name>
+  <version>0.0.0</version>
+  <description>Description of @package_name</description>
+  <maintainer email="@maintainer_email">@maintainer_name</maintainer>
+  <license>Apache License 2.0</license>
```

Comment:
as the command line tool takes a license: the user provided license should be expanded here rather than hard-coding Apache 2 (same in the other template below)